modin/experimental/core/execution/native/implementations/omnisci_on_native/io/io.py

```diff
@@ -466,7 +466,9 @@ def _read_csv_check_support(
             )
 
         parse_dates_unsupported = isinstance(parse_dates, dict) or (
-            isinstance(parse_dates, list) and isinstance(parse_dates[0], list)
+            isinstance(parse_dates, list)
+            and len(parse_dates) > 0
```

Comment:
Does this mean that if `parse_dates` is an empty list, we'll just not parse dates silently? Is that ideal?